packages/lib/CalendarService.ts

```diff
@@ -47,6 +47,17 @@ const getDuration = (start: string, end: string): DurationObject => ({
   minutes: dayjs(end).diff(dayjs(start), "minute"),
 });
 
+const buildUtcOffset = (minutes: number): string => {
+  const h =
+    minutes > 0
+      ? "+" + (Math.floor(minutes / 60) < 10 ? "0" + Math.floor(minutes / 60) : Math.floor(minutes / 60))
+      : "-" +
+        (Math.ceil(minutes / 60) > -10 ? "0" + Math.ceil(minutes / 60) * -1 : Math.ceil(minutes / 60) * -1);
+  const m = minutes > 0 ? minutes % 60 : (minutes % 60) * -1;
```

Comment:
Correct me if I'm wrong but wouldn't a minutes offset like this cause a potential for:
```
-2:-15
// in case of -135?
```

Reply:
You're right, but yeah abs is a little neater anyway.

Reply:
Oh yeah, for sure. Way better. I believe there’s still room for code improvement but I’ll do that in a follow up where I take care of the daylight monster 🥲